packages/core/src/external/carequality/dr/validating-dr.ts

```diff
@@ -0,0 +1,56 @@
+import { DocumentRetrievalReqFromExternalGW, DocumentReference } from "@metriport/ihe-gateway-sdk";
+import {
+  METRIPORT_HOME_COMMUNITY_ID,
+  METRIPORT_REPOSITORY_UNIQUE_ID,
+  validateBasePayload,
+  extractDocumentUniqueId,
+} from "../shared";
+import { S3Utils } from "../../aws/s3";
+import { Config } from "../../../util/config";
+import { XDSRegistryError } from "../error";
+
+const region = Config.getAWSRegion();
+const medicalDocumentsBucketName = Config.getMedicalDocumentsBucketName();
+
+export async function validateDR(
```

Comment:
This function does much more than validate a DR... let's be careful how we rename functions!